src/github.ts

```diff
@@ -0,0 +1,59 @@
+import { Octokit } from '@octokit/rest';
+
+/**
+ * @summary Get CHANGELOG.yml
+ * @function
+ * @public
+ *
+ * @param {string} owner - repo owner
+ * @param {string} repo - repo
+ * @param {octokit} octokit - Octokit client
+ * @param {branch} branch - the branch where file exists
+ * @returns {Promise<any>} The blob as returned by GitHub API
+ *
+ * Get the CHANGELOG.yml file using the trees and blob APIs. getContent API
+ * does not allow for files bigger than 1MB so we need to get the sha and fetch it.
+ */
+export const getChangelogYML = async (
+	owner: string,
+	repo: string,
+	branch: string,
+	octokit: Octokit,
+): Promise<any> => {
```

Comment:
I assume `blobResponse.data.content` is a buffer?
```suggestion
): Promise<Buffer> => {
```
But if not then presumably it's a string:
```suggestion
): Promise<string> => {
```

Reply:
It's a string @Page- , done!